index.html

```diff
@@ -39,8 +39,209 @@ <h1 class="hero-title" >I’m Mariane <br>
+                    <h2 class="work-title" >Multi-Post Stories</h2>
+                    <ul class="work-list">
+                        <h4>CANOPY</h4>
+                        <li class="work-list-item" >
+                            <img src="./assets/Counter.svg" alt="">
+                            Back End Dev
+                        </li>
+                        <li class="work-list-item" >
+                            <img src="./assets/Counter.svg" alt="">
+                            2015
+                        </li>
+                    </ul>
+                    <article class="work-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </article>
+                    <ul class="work-tech">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul>
+                    <article class="work-button ">
+                        <button class="btn" > See project </button>
+                    </article>
+                </aside>
+            </article>
+        </div>
+    </section>
+
+    <section class="hero hero-about">
+        <div class="container-big">
+            <article class="hero-content separete-items-down">
+                <h1 class="hero-title" >About <br>
+                    Myself <br></h1>
+                <p class="hero-text">
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+
+                <a href="#">LET'S CONNECT</a>
+
+                <aside class="social-media">
+                    
+                    <a href="">
+                        <img src="./assets/social media icons.svg" alt="">
+                    </a>
+                </aside>
+
+                <article class="work-button work-button-about">
+                    <button class="btn" > Get my resume </button>
+                </article>
+            </article>
+            
+        </div>
+        <article class="languajes">
+            
+            <div class="languajes-title-cont">
+                <h2>Languages</h2>
+                <img src="/assets/arrow-down.svg" alt="">
+            </div>
+
+            <ul class="languajes-cont">
+                <li class="languaje-item" >
+                    <img src="./assets/Ellipse 1.svg" alt="">
+                    <p>Javascript</p>
+                </li>
+                <li class="languaje-item" >
+                    <img src="./assets/Ellipse 2.svg" alt="">
+                    <p>HTML</p>
+                </li>
+                <li class="languaje-item" >
+                    <img src="./assets/Ellipse 3.svg" alt="">
+                    <p>CSS</p>
+                </li>
```

Comment:
[OPTIONAL] Kindly consider adding values to all the alt attributes of your image to help provide alternative for screen readers and also serve as a support when the image links do not work well. :wink: